tgi-entrypoint.sh

```diff
@@ -2,4 +2,5 @@
 
 ldconfig 2>/dev/null || echo 'unable to refresh ld cache, not a big deal in most cases'
 
+source ./server/.venv/bin/activate
```

Comment:
Maybe instead the full path to not assume what's the `WORKDIR` set to?

```suggestion
source /usr/bin/server/.venv/bin/activate
```

Reply:
I made that on purpose. I want it to be linked to the workdir, because that's how `uv` works.